src/uiSelectDirective.js

```diff
@@ -411,7 +411,7 @@ uis.directive('uiSelect',
             }
 
             // Reset the position of the dropdown.
-            dropdown[0].style.opacity = 0;
+            dropdown[0].classList.add('ui-select-detached');
```

Comment:
No it's new class, I haven't added it to templates so it's here first time and we see addition on two sides of `if ($select.open)` so it's either one or another.
 And even if class was added already it will not duplicase as `classList.add` adds only if not present.